File: project_euler/Problem028/Problem028.js
function Problem028() {
    let target = 1001
    let steps = target >>> 1
    let result = (steps * steps * steps * 16 + steps * 26) / 3 + steps * steps * 10 + 1
    console.log(result)
}

module.exports = {
    Problem028 : Problem028
};

File: project_euler/Problem028/Problem028.cpp
#include <iostream>
using namespace std;

class Problem028 {
public:
    void solution() {
        int target = 1001;
        int steps = target >> 1;
        long long result = (long long) (steps * steps * steps * 16 + steps * 26) / 3 + steps * steps * 10 + 1;
        cout << result << endl;
    }
};
